db/schema.rb

```diff
@@ -66,6 +66,8 @@
     t.string "verification_code"
     t.bigint "pair_id", null: false
     t.string "messenger"
+    t.boolean "is_phone_enabled", default: true
+    t.string "facebook", default: "facebook.com/zuck"
```

Comment:
Nigdy nie modyfikuj schema.rb ręcznie. Ten plik służy m.in wygodzie programistów żeby mogli zobaczyć co jest w bazce ale nie powinien być ręcznie modyfikowany. Jest on generowany na podstawie plików migracji i odwzorowuje stan bazy.  Kiedy robimy deploy puszczamy `rails db:migrate` . Ta komenda patrzy jakie migracje są nowe i wykonuje tylko je. W ten sposób można dodać pola do bazy bez jej całkowitego zniszczenia i usunięcia danych.

aby stworzyć nową migrację wpisujemy w konsolę
`rails g migration <nazwa>`. Zrobiłem to za ciebie już ;)

Ps. pliki migracji znajdują sie w katalogu `db/migrate`